engine/client/s_vox.c

```diff
@@ -545,7 +545,7 @@ void VOX_ParseLineCommands( char *pSentenceData, int sentenceIndex )
 		length = pNext - pSentenceData;
 		if( tempBufferPos + length > sizeof( tempBuffer ))
 		{
-			Con_Printf( S_ERROR "sentence too long!\n" );
+			Con_Printf( S_ERROR "Sentence exceeded max length of %d\n", sizeof(tempBuffer) - 1 );
```

Comment:
"Too long" is better for not english-speaking users. Maybe better "Sentence too long (%d bytes max)\n"